src/components/tutorialEditor/index.tsx

```diff
@@ -135,12 +307,148 @@ const TutEditor: React.FC = (): JSX.Element => {
+                            }}
+                          />
+                          <Arrow
+                            icon={chevronDown}
+                            onClick={(): void => {
+                              changeOrder("down", index);
+                            }}
+                          />
+                        </UpDownContainer>
+                        <TextInput
+                          value={content[index].value}
+                          style={{
+                            display:
+                              content[index].type === "text" ? "block" : "none"
+                          }}
+                          type="text"
+                          placeholder="Type text here..."
+                          onChange={(
+                            event: React.ChangeEvent<HTMLInputElement>
+                          ): void => {
+                            handleTextChange(index, event);
+                          }}
+                        />
+                        <AceEditor
+                          value={content[index].value}
+                          style={{
+                            position: "relative",
+                            marginTop: "0px",
+                            height: "82px",
+                            width: "100%",
+                            backgroundColor: background,
+                            fontFamily: "Source Code Pro",
+                            display:
+                              content[index].type === "code" ? "block" : "none"
+                          }}
+                          fontSize="16px"
+                          mode="python"
+                          theme="tomorrow_night_eighties"
+                          placeholder="Type code here..."
+                          setOptions={{
+                            enableBasicAutocompletion: true,
+                            enableLiveAutocompletion: true,
+                            enableSnippets: true,
+                            tabSize: 4
+                          }}
+                          onChange={(code: string): void => {
+                            handleCodeChange(index, code);
+                          }}
+                        />
+                      </ContentBlockInnerWrapper>
+                      <DeleteBlock
+                        visible={hovering === index ? "flex" : "none"}
+                        onClick={(): void => {
+                          deleteBlock(index);
+                        }}
+                      >
+                        <TrashIcon icon={bxTrashAlt} />
+                      </DeleteBlock>
+                    </ContentBlock>
+                  );
+                })}
+              </StepContentList>
+              <StyledButton
+                onClick={addBlock}
+                style={{ width: 150, userSelect: "none" }}
+              >
+                Add Block
+              </StyledButton>
```

Comment:
There's already a button in `src/components/textEditor/button.tx` so if you could make that into it's own component, we should we able to avoid repeated code.

Reply:
Now that I think about it you might need to refactor that button to accept a background color as a prop instead of current janky implementation.